charts/karpenter/values.yaml

```diff
@@ -166,6 +166,10 @@ settings:
   batchIdleDuration: 1s
   # -- AWS-specific configuration values
   aws:
+    # -- Role to assume for calling AWS services.
+    assumeRoleARN: ""
+    # -- Duration of assumed credentials in minutes. Default value is 15 minutes. Not used unless aws.assumeRoleARN set.
+    assumeRoleDuration: ""
```

Comment:
Just curious: is there a reason that we have chosen to not just make this `roleDuration` so that it can be generalized if someone wants to extend their role duration outside of the context of assume role?

Reply:
Reference: https://github.com/aws/karpenter/pull/4370#discussion_r1283807940